test/functional/p2p_segwit.py

```diff
@@ -930,11 +931,12 @@ def test_tx_relay_after_segwit_activation(self):
         raw_tx = self.nodes[0].getrawtransaction(tx3.hash, 1)
         assert_equal(int(raw_tx["hash"], 16), tx3.calc_sha256(True))
         assert_equal(raw_tx["size"], len(tx3.serialize_with_witness()))
-        vsize = (len(tx3.serialize_with_witness()) + 3*len(tx3.serialize_without_witness()) + 3) / 4
-        assert_equal(raw_tx["vsize"], vsize)
+        weight = len(tx3.serialize_with_witness()) + 3*len(tx3.serialize_without_witness())
+        assert_equal(raw_tx["vsize"], math.ceil(weight / 4))
+        assert_equal(raw_tx["weight"], weight)
         assert_equal(len(raw_tx["vin"][0]["txinwitness"]), 1)
         assert_equal(raw_tx["vin"][0]["txinwitness"][0], hexlify(witness_program).decode('ascii'))
-        assert(vsize != raw_tx["size"])
+        assert(weight / 4 != raw_tx["size"])
```

Comment:
I think this can stay as it was (if you add a `vsize` variable above).

assert is a statement, not a function, so if you're going to touch this line, then can you change to:

```
assert weight / 4 != raw_tx["size"]
```